src/Controls/tests/Core.UnitTests/StrokeShapeTests.cs

```diff
@@ -127,5 +127,18 @@ public void TestEllipseConstructor(string value)
 
 			Assert.NotNull(ellipse);
 		}
+
+		[Theory]
+		[InlineData("20")]
+		public void TestRoundRectangleSingleValue(string value)
```

Comment:
Could you include also tests validating `border-color` and `border-width`?

Reply:
I couldn't find any existing tests for these types of attributes for the other controls. Could you perhaps point me to some of these? For example, I don't see a test where CSS `padding: 4` is tested to actually end up on the control. I think the CSS value is typically just mapped one to one to the `Padding `property internally. 

The color in this setup is set through the existing `BrushTypeConverter` which is covered by tests, whereas the width value is mapped 1-to-1 to the `StrokeWidth `property internally as mentioned above.